README.md

```diff
@@ -1,15 +1,64 @@
 # Add a Syncfusion Blazor Progress Button to a Blazor WebAssembly App
 
-A quick start Blazor project that shows how to create and configure the Syncfusion Blazor Progress Button component in a Blazor WebAssembly app. This project also includes a code snippet to add icons, change spinner positions, and how to animate the button content and the progress button spinner.
-
-Example: https://blazor.syncfusion.com/demos/buttons/progress-button 
-
-Documentation: https://blazor.syncfusion.com/documentation/progress-button/getting-started
+The [Blazor Progress Button](https://www.syncfusion.com/blazor-components/blazor-progress-button?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples) is a graphical user interface component used to visualize the progression of an operation in the background. A quick start Blazor project that shows how to create and configure the Syncfusion Blazor Progress Button component in a Blazor WebAssembly app. This project also includes a code snippet to add icons, change spinner positions, and how to animate the button content and the progress button spinner. 
 
 ## Project pre-requisites
 Make sure that you have the compatible versions of Visual Studio 2019 or 2022 and .NET Core SDK latest version in your machine before starting to work on this project.
 
 ## How to run this application?
 To run this application, you need to first clone the how-to-add-syncfusion-blazor-progress-button-to-blazor-webassembly-app repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
 
+## Features and benefits
+
+## Progress Button Icons
+
+Users tend to perceive visual information better than text. The Blazor progress button has built-in options to include an icon to the right, left, top, and bottom of a text or provide an icon-only button.
+
+## Predefined styles
+
+The Blazor progress button has the following predefined styles to be used based on the application scenario:
+
+* Primary
+* Success
+* Info
+* Warning
+* Danger
+
+## Custom Progress Button
+
+Blazor Progress buttons can be customized in multiple ways. You can choose from different button sizes, default and small, as required in your application.
+
+## Related links
+[Learn More about Blazor ListView](https://www.syncfusion.com/blazor-components/blazor-progress-button?utm_source=github&utm_medium=listing&utm_campaign=blazor-progress-button-github-samples)
```

Comment:
is this ListView control?
![image](https://user-images.githubusercontent.com/61148953/173589850-1b8fb598-56c9-44c9-a996-5a66609348d5.png)

Reply:
resolved